server/api/order.js

```diff
@@ -18,40 +68,106 @@ router.get("/:orderId", async (req, res, next) => {
     const order = await Order.findByPk(req.params.orderId, {
       include: OrderDetails,
     });
+    console.log("order:", order);
+
     res.json(order);
   } catch (err) {
     next(err);
   }
 });
 
-// router.post("/", async (req, res, next) => {
-//   try {
-//     res.json(
-//       await Order.create(req.body, {
-//         include: OrderDetails,
-//       })
-//     );
-//   } catch (err) {
-//     next(err);
-//   }
-// });
+router.get("/:orderId/:orderDetailsId", async (req, res, next) => {
+  try {
+    const order = await OrderDetails.findByPk(req.params.orderDetailsId);
+    res.json(order);
+  } catch (err) {
+    next(err);
+  }
+});
 
-router.post("/", async (req, res, next) => {
+router.post("/user/:userId/:productId", async (req, res, next) => {
```

Comment:
I think normally with a post route, you would just include these ids in the body